### PR TITLE
Fix cli build param ver

### PR DESCRIPTION
**Pull request by IgorGov**

### CODE REVIEW

**Review by gadotroee — APPROVED** [review]

File: cli/Makefile [hunk 1]
```diff
@@ -23,7 +23,7 @@ build: ## Build mizu CLI binary (select platform via GOOS / GOARCH env variables
 					   -X 'github.com/up9inc/mizu/cli/mizu.Branch=$(GIT_BRANCH)' \
 					   -X 'github.com/up9inc/mizu/cli/mizu.BuildTimestamp=$(BUILD_TIMESTAMP)' \
 					   -X 'github.com/up9inc/mizu/cli/mizu.Platform=$(SUFFIX)' \
-					   -X 'github.com/up9inc/mizu/cli/mizu.SemVer=$(SEM_VER)'" \
+					   -X 'github.com/up9inc/mizu/cli/mizu.Ver=$(SEM_VER)'" \
```
Comment (RoyUP9) [inline cli/Makefile:26]: SEM_VER variable is 0.0.0, we want it to stay this way?

File: Dockerfile [hunk 2]
```diff
@@ -75,7 +75,7 @@ RUN go build -ldflags="-extldflags=-static -s -w \
     -X 'github.com/up9inc/mizu/agent/pkg/version.GitCommitHash=${COMMIT_HASH}' \
     -X 'github.com/up9inc/mizu/agent/pkg/version.Branch=${GIT_BRANCH}' \
     -X 'github.com/up9inc/mizu/agent/pkg/version.BuildTimestamp=${BUILD_TIMESTAMP}' \
-    -X 'github.com/up9inc/mizu/agent/pkg/version.SemVer=${SEM_VER}'" -o mizuagent .
+    -X 'github.com/up9inc/mizu/agent/pkg/version.Ver=${SEM_VER}'" -o mizuagent .
```
Comment (RoyUP9) [inline Dockerfile:78]: SEM_VER variable is 0.0.0, we want it to stay this way?

File: agent/pkg/version/consts.go [hunk 3]
```diff
@@ -1,7 +1,7 @@
 package version
 
 var (
-	SemVer         = "0.0.1"
+	Ver         = "0.0.1"
```
Comment (gadotroee) [inline agent/pkg/version/consts.go:4]:
```suggestion
	Ver         = "0.1"
```

File: cli/Makefile (hunk 1 above)
Comment (gadotroee) [reply cli/Makefile:26]: it should be changed to 0.0 or 0.1 IMO

**Review by gadotroee — CHANGES_REQUESTED** [review]
small changes

**Review by RoyUP9 — APPROVED** [review]

File: Dockerfile [hunk 4]
```diff
@@ -67,15 +67,15 @@ COPY agent .
 ARG COMMIT_HASH
 ARG GIT_BRANCH
 ARG BUILD_TIMESTAMP
-ARG SEM_VER=0.0.0
+ARG VER=0.0.0
```
Comment (gadotroee) [inline Dockerfile:70]:
```suggestion
ARG VER=0.0
```

File: agent/kratos/build-push-featurebranch.sh [hunk 5]
```diff
@@ -7,9 +7,9 @@ SERVER_NAME=mizu-kratos
 GIT_BRANCH=$(git branch | grep \* | cut -d ' ' -f2 | tr '[:upper:]' '[:lower:]')
 
 DOCKER_REPO=$REPOSITORY/$SERVER_NAME/$GIT_BRANCH
-SEM_VER=${SEM_VER=0.0.0}
+VER=${VER=0.0.0}
```
Comment (gadotroee) [inline agent/kratos/build-push-featurebranch.sh:10]:
```suggestion
VER=${VER=0.0}
```

Comment (IgorGov) [reply agent/kratos/build-push-featurebranch.sh:10]: not related

**Review by gadotroee — APPROVED** [review]

**Pull request merged**